languages/python/exercises/concept/chaitanas-colossal-coaster/.docs/introduction.md

```diff
@@ -64,26 +55,21 @@ print(ex_list)
 #=> [1, 2, 3, 5, 6, 7, 8, 9]
 ```
 
-# Removing Items from the list
-
-Topics to discuss:
+### Removing Items
 
-1. `remove`
-2. `pop`
-3. `clear`
-
-If you want to remove an item from a list, you can use the `remove` function which takes one parameter. Feed the function with the item you want to remove from the list. the `remove` function will throw a `ValueError` if the item does not exist in the loop.
+If you want to delete an element, you can use `.remove()` and pass the item you want removed from the list. `.remove()` will throw a `ValueError` if the item to be removed is not in the list.
 
 ```python
-ex_list = [1, 2, 3]
-ex_list.remove(2)
-print(ex_list)
-#=> [1, 3]
-ex_list.remove(0)
-#=> ValueError: list.remove(x): x not in list
+>>> ex_list = [1, 2, 3]
+>>> ex_list.remove(2)
+>>> print(ex_list)
+[1, 3]
+
+>>> ex_list.remove(0)
+ValueError: list.remove(x): x not in list
 ```
 
-There is another way to remove an item from the list. If you know the index of the item which you want to remove, you can use `pop` function. Pop takes 1 parameter, the index of the item you need to remove, and return it to you. If you specify an index more than the length of the list, you will get an `IndexError`. If you dont specify an index, the function will remove the last element and return it to you.
+Conversly, using the `.pop()` function will both remove **and** `return` an element for use. `.pop()` takes 1 parameter -- the index of the item you need to remove and recieve. If you specify an index number higher than the length of the list, you will get an `IndexError`. If an index is not specified, the last element of the list will be removed and returned to you.
```

Comment:
Rather/Alternatively (Conversely)

Reply:
@valentin-p  -- `Alternatively` is a much better word.  Thank you!